Makefile

```
@@ -343,14 +343,26 @@ define PRINT_REQUEST_MANAGER_URL
 endef
 
 define PRINT_INTEGRATION_DISPATCHER_URL
-	$(call PRINT_SERVICE_URLS,Integration Dispatcher,$(INGRESS_PREFIX)-integration-dispatcher, \
+	@echo "--- Your Integration Dispatcher URLs are: ---"
+	@sleep 10
+	@EXTERNAL_HOST=$$($(call GET_EXTERNAL_HOST,$(INGRESS_PREFIX)-integration-dispatcher-slack)); \
+	if [ -z "$$EXTERNAL_HOST" ]; then \
+		EXTERNAL_HOST=$$($(call GET_EXTERNAL_HOST,$(INGRESS_PREFIX)-integration-dispatcher)); \
+	fi; \
+	if [ -n "$$EXTERNAL_HOST" ]; then \
+		echo "  OpenAPI Schema: https://$$EXTERNAL_HOST/docs"; \
+		echo "  Health: https://$$EXTERNAL_HOST/health"; \
```

maybe we don't need the route name change? also, certain endpoints will now only be accessible internally.
```suggestion
	@echo "--- Your Integration Dispatcher URLs are: ---"
	@sleep 10
	@EXTERNAL_HOST=$$($(call GET_EXTERNAL_HOST,$(INGRESS_PREFIX)-integration-dispatcher)); \
	if [ -z "$$EXTERNAL_HOST" ]; then \
		EXTERNAL_HOST=$$($(call GET_EXTERNAL_HOST,$(INGRESS_PREFIX)-integration-dispatcher)); \
	fi; \
	if [ -n "$$EXTERNAL_HOST" ]; then \
		echo "  OpenAPI Schema: http://$(MAIN_CHART_NAME)-integration-dispatcher.$(NAMESPACE).svc.cluster.local/docs"; \
		echo "  Health: http://$(MAIN_CHART_NAME)-integration-dispatcher.$(NAMESPACE).svc.cluster.local/health"; \
```

---

@mhdawson thoughts on this?